scripts/bootstrap-aio.yml

```diff
@@ -36,6 +36,28 @@
 - name: Execute the OSA AIO bootstrap
   include: "{{ lookup('env', 'OA_DIR') }}/tests/bootstrap-aio.yml"
   vars:
```

Comment:
I've proposed an upstream change to move these variables to the ``vars`` section:
https://review.openstack.org/#/c/428820/1